workers/ohsome_quality_analyst/oqt.py

```diff
@@ -250,10 +287,7 @@ async def _create_report(
 
 
 @create_report.register
-async def _create_report(  # noqa
-    parameters: ReportBpolys,
-    force: bool = False,
-) -> Report:
+async def _create_report(parameters: ReportBpolys, *_args) -> Report:  # noqa
```

Comment:
see above

Reply:
There is. Added exact flake8 warning code.